### ISSUE TITLE
Pop-up Text is not announced by Screen Readrs on Know More Button in See Events on Events Page

### ISSUE THREAD

**Issue by VarunKumar-Xaviers** [xxchanjotxx/malhar-live#10]
The Pop-up that appears on screen after the user clicks on the Know More button, The text that is displayed is not announced by the screen reader. 

There is also no keyboard focus for the Close button so there is no way for a keyboard user to close the dialog box once opened.

**Comment by VarunKumar-Xaviers** [comment]
We have to use the following code to fix this in the function call of the know more button 
`document.querySelector( '[tabindex="-1"]' ).focus();` or `document.getElementById( "focus" ).focus();` 

- change the para attribute  to the code given below 
`<p id="focus" tabindex="-1"></p>`

**Comment by VarunKumar-Xaviers** [comment]
Made a temporary work around for the focus on paragraph
